a11y.js

```diff
@@ -53,6 +53,11 @@ const pageTypes = {
     '/html/head/iframe',
     "//div[@id='root']/header/nav/div/div[1]/div/ul",
   ],
+  storyPage: [
+    "//div[@id='root']/header/nav/div/div[1]/div/ul",
+    '/iframe', // known issue above with iframes should be revisited once https://github.com/bbc/bbc-a11y/pull/313 gets merged.
+    "//div[@id='root']/div/div[1]/main/div[37]/div/div/div", // issue with IDT2 includes
```

Comment:
not sure how we'd go around this as we have no control over the include markup but the only option for now would be to hide this error in a much better way than this. Perhaps give the include container div a specific class name and hide using that class name, if so would we want that handled in another issue ?

Reply:
![Screen Shot 2020-05-12 at 12 50 35](https://user-images.githubusercontent.com/25608386/81694078-cae10980-9458-11ea-8888-b09bb2fae251.png)

Reply:
All includes are wrapped in a `div`, I'd label it up with a class as you suggest @rhenshaw56 to make this cleaner. Don't see any reason that can't be in this PR.

Reply:
@andrewscfc I looked into this and I don't think it could be done in the way I thought originally. We can only hide errors with a pattern matching the failing paths returned by `bbc-a11y` eg `//div[@id='root']/div/div[1]/main/div[37]/div/div/div/h2`, `//div[@id='root']/div/div[1]/main/div[37]/div/div/div/span` after each run but we can't add in a custom class selector. I reckon since we would have control over whatever asset we run `bbc-a11y` with we can always limit it's failure on ci, however we would be dealing with the fact that we do have some includes that's not a11y complaint on some of our story pages.

Reply:
Thanks for trying the class approach, I'm happy with this as it is 👍